pypasta/Config.py

```diff
@@ -195,6 +196,10 @@ def path(name):
             mbox_raw = mbox['raw']
             mbox_pub_in = mbox['pubin']
 
+            mbox_pw = 'patchwork'
+            if mbox_pw in mbox:
+                self.pw = Patchwork(mbox[mbox_pw]['url'])
```

Comment:
These three lines will be completely modified two patches above, can we somehow use the eventual version of this change in this patch?

Reply:
Why does the instance need to be a member of Config? Wouldn't the mbox subsystem (e.g., repo.mbox.patchwork) be a better place?